src/EnergyPlus/HVACInterfaceManager.cc

```diff
@@ -128,12 +127,13 @@ void UpdateHVACInterface(EnergyPlusData &state,
             totDemandSideMaxAvail += state.dataLoopNodes->Node(demInNode).MassFlowRateMaxAvail;
         }
         TmpRealARR = state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue;
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue(1) =
+        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue[0] =
             std::abs(totDemandSideMassFlow - state.dataLoopNodes->Node(InletNode).MassFlowRate);
-        state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue({2, DataConvergParams::ConvergLogStackDepth}) =
-            TmpRealARR({1, DataConvergParams::ConvergLogStackDepth - 1});
-        if (state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue(1) > DataConvergParams::HVACFlowRateToler) {
-            state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACMassFlowNotConverged(1) = true;
+        for (int logIndex = 2; logIndex <= DataConvergParams::ConvergLogStackDepth; logIndex++) {
+            state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue[logIndex] = TmpRealARR[logIndex - 1];
+        }
+        if (state.dataConvergeParams->AirLoopConvergence(AirLoopNum).HVACFlowDemandToSupplyTolValue[0] > DataConvergParams::HVACFlowRateToler) {
```

Comment:
There is a penny-wise pound-foolish thing about calling a namespace `DataConvergParams`. If you are so concerned about shortening the name, remove the word `Data` rather than the `e` from Converge`. While on the topic, the word `Data` should be removed from all names, namespaces, types, variables. IT'S ALL DATA! What information does the word `Data` add? I'm not joking.

Reply:
It points me to all the files with global parameters. If Data is removed, put them in a new \Something folder. It just seems these files should be grouped together.

Reply:
Hmmmmm. So, there is no one-to-one correspondence between files and namespaces (obviously, since all files in the repo are in an outer EnergyPlus namespace). You can keep the file name as DataX.hh or whatever and shorten the namespace if that helps.  Does it?

In general, EnergyPlus has too many namespaces and they are all way too long (this is probably an artefact of the automatic translation which created the 1-to-1 file/namespace mapping). We should have something like 15-20 namespaces and they should have short names like `Surface`, `HVAC`, `Plant`, etc. Look at library namespaces, the standard container library namespace is `std` not `StandardContainer`. There is a page about this in the wiki also.

Reply:
You can leave `Data` in the file name and only change the `namespace` name. The `namespace` does not have to have the same name as the file. It doesn't matter that much, it's just that these really long `namespace` names make the code more difficult to read than it would otherwise be (for me at least).

Reply:
I also avoided this aspect for now, but can start changing that here if desired.  I was just trying to create a small-ish PR to clean up the interface manager, and I don't want the scope to creep too much, but I also understand that these are opportune times to make these changes when we are already refactoring small file groups.

Reply:
Mostly they are opportune times for me to make comments. Put a pin in it if you want.